packages/@aws-cdk/aws-ec2/test/vpc.test.ts

```diff
@@ -417,17 +417,49 @@ nodeunitShim({
       test.done();
     },
 
-    'natGateways = 0 requires there to be no PRIVATE subnets'(test: Test) {
+    'natGateways = 0 requires there to be no configured PRIVATE subnets'(test: Test) {
```

Comment:
suggest renaming this to be clearer
```suggestion
    'natGateways = 0 throws if PRIVATE subnets are configured'(test: Test) {
```